frigate/models.py

```diff
@@ -18,22 +18,33 @@ class Event(Model):  # type: ignore[misc]
     camera = CharField(index=True, max_length=20)
     start_time = DateTimeField()
     end_time = DateTimeField()
-    top_score = FloatField()
-    score = FloatField()
+    top_score = (
+        FloatField()
+    )  # TODO remove when columns can be dropped without rebuilding table
```

Comment:
top_score, score, region, box are all populated in the db. They dont come back from the /events api endpoint because we only grab specific columns, but they are there.

Reply:
made this comment before i saw the changes down below.